### DISCUSSION TITLE
Built in tool that can make a row of timing points in a selected timeline

### DISCUSSION THREAD

**Discussion by akiteruakashi** [ppy/osu#16379] [Ideas] [answered]
Some maps (especially taiko) have a bunch of inherited timing points in a part of a song (mostly a build-up before a beat drop).
![image](https://user-images.githubusercontent.com/26337426/148674797-bf50c6f9-e46c-40e5-9b1f-2db364837b6e.png)
this makes the map goes well with the song thus making it interesting and fun.

one way to achieve this is by adding timing points one by one which is pretty time consuming, let alone changing the SV value.
I don't know if there is an existing program that can make this but I think a built-in tool would be convenient.

It could be useful for taiko mappers to have a tool that can make a line of timing points in a selected timeline that can get higher or lower depending on the mapper's choice without having to do it manually.

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/discussions/13141 (i think?)

**Reply by peppy** [reply]
Yep.
